scripts/linux/bootstrap-kubernetes-minion.sh

```diff
@@ -1,14 +1,15 @@
 #!/bin/sh
 
+SCRIPT_PATH="/vagrant/scripts/linux"
```

Comment:
Let me think if we can get rid of this hardwired path...

Reply:
ok, see my comment at the end